src/components/Sidebar.tsx

```diff
@@ -7,17 +7,17 @@ const Sidebar: FC = () => (
   <nav className="sidebar">
     <ul className="sidebar-list">
       <li className="sidebar-item">
-        <NavLink exact to="/" activeClassName="active" title="New Snippet">
+        <NavLink to="/" className={({ isActive }) => isActive ? "active" : ""} title="New Snippet" end>
```

Comment:
Since our navlinks are always active, do we even need this change?

Also, what's that "end" at the end?